README.org

```diff
@@ -30,6 +30,15 @@ To install the package, you can use any Emacs package manager of your choice, to
 
 Then, you can simply call ~scala-ts-mode~ in a Scala buffer.
 
+* Configuration
+You may wish to customize `treesit-font-lock-level`, using `M-x customize-variable` or by adding a line such as this to your `~/.emacs.el`:
```

Comment:
Can we change the `~/.emacs.el` to `your user init file`? Since a lot of people now actually using the `.config/emacs/init.el` file. Or better yet, we can change it to `user-init-file` which you can find by executing `C-h v user-init-file RET` in a running Emacs instance?

Or if not, we can simply replace it with init file instead of a specific file and let the user figure out which one they are using 🤔

Reply:
good point. adjusted to just say "init file"